VmAgent.Core/VmDirectories.cs

```diff
@@ -41,9 +41,9 @@ public VmDirectories(string rootPath)
 
             AgentStateFile = Path.Combine(PlayFabFolderOnPrimaryDrive, "PlayFabVmAgentState");
             AgentStateTempFile = Path.Combine(PlayFabFolderOnPrimaryDrive, "PlayFabVmAgentState.tmp");
-            MonitoringStateFile = Path.Combine(TempStorageRootVm, "PlayFabMonitoringState");
-            MonitoringStateTempFile = Path.Combine(TempStorageRootVm, "PlayFabMonitoringState.tmp");
-            HostConfigOverrideFile = Path.Combine(TempStorageRootVm, "HostConfigOverride.json");
+            MonitoringStateFile = Path.Combine(PlayFabFolderOnPrimaryDrive, "PlayFabMonitoringState");
+            MonitoringStateTempFile = Path.Combine(PlayFabFolderOnPrimaryDrive, "PlayFabMonitoringState.tmp");
```

Comment:
Why are these files being moved out of the temporary folder? Especially this .tmp file--i would expect that to be in a temporary folder

Reply:
wrong commit included by mistake, removed, thanks!